### ISSUE TITLE
Populate where blueprint not showing entire result

### ISSUE THREAD

**Issue by adavia** [balderdashy/sails#2882]
So lets say i have this 2 models, User and Pictures. For some reason when i want to use the populate where blueprint  GET /user/1/pictures, it only shows up those populates values and not the entire user model with it pictures. So instead of something like:

{
  "name": "Andrew,
  "age": 47,
  "pictures": [ {
    "name": "picture1.jpg",
    "id": 1,
    "createdAt": "2012-05-14T01:21:05.000Z",
    "updatedAt": "2013-01-15T01:18:40.000Z"
  },
   {
    "name": "picture2.jpg",
    "id": 2,
    "createdAt": "2012-05-14T01:21:05.000Z",
    "updatedAt": "2013-01-15T01:18:40.000Z"
  } ],
  "createdAt": "2013-10-14T01:22:00.000Z",
  "updatedAt": "2013-10-15T01:20:54.000Z"
}

I only get

"pictures": [ {
    "name": "picture1.jpg",
    "id": 1,
    "createdAt": "2012-05-14T01:21:05.000Z",
    "updatedAt": "2013-01-15T01:18:40.000Z"
  },
   {
    "name": "picture2.jpg",
    "id": 2,
    "createdAt": "2012-05-14T01:21:05.000Z",
    "updatedAt": "2013-01-15T01:18:40.000Z"
  } ]

This is not what the documentation shows, so im wondering what could it be wrong?

**Comment by kfirstri** [comment]
Could you paste your controller or link to your repository?

**Comment by mikermcneil** [comment]
Thanks for posting, @adavia.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
